packages/block-library/src/block/edit.js

```diff
@@ -59,39 +62,38 @@ export default function ReusableBlockEdit( {
 		ref
 	);
 
-	const alignments = [ 'wide', 'full' ];
-
 	useEffect( () => {
```

Comment:
I wonder about something can we avoid setting the "layout" attribute? and use `useMemo` for both `inferredAlignment` and `inferredLayout`. To avoid messing up with re-renderings and undo/redo as well.